<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachScheduleDialog</class>
 <widget class="QDialog" name="AttachScheduleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Schedule Backup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#AttachScheduleDialog {
	background-color: rgb(6, 134, 103);
}

QFrame {
	background: white;
}

QPushButton {
	padding: 16px 32px;
	font-size: 32px;
	font-weight: 500;
	qproperty-iconSize: 36px 36px;
	background-color: rgb(6, 134, 103);
	color: white;
	text-align: left;
}

QRadioButton {
	padding: 32px;
	font-size: 24px;
	font-weight: 500;
	background-color: white;
	color: rgb(29, 31, 39);
	border: 1px solid rgb(29, 31, 39);
}

QRadioButton::indicator {
	width: 0;
	height: 0;
}

QRadioButton:hover {
	background-color: rgb(234, 234, 234);
}

QRadioButton:checked {
	color: white;
	background-color: rgb(6, 134, 103);
	border: none;
}

QProgressBar {
    border: 1px solid rgb(29, 31, 39);
    height: 20px;
    text-align: center;
    font-weight: bold;
}

QProgressBar::chunk {
    background-color: rgb(6, 134, 103);
}

QLabel {
    color: rgb(29, 31, 39);
    font-weight: bold;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">* {
	font-family: &quot;Ubuntu Sans&quot;;
}

QFrame{
	border: none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	font-size: 16pt;
	color: rgb(6, 134, 103);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Attach Schedule</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="styleSheet">
         <string notr="true">QFrame {
	background: rgb(234, 234, 234);
}

QLabel {
	background-color: rgb(234, 234, 234);
	font-weight: 500;
	font-size: 16pt;
}

QLineEdit {
	color: rgb(6, 134, 103);
	font-weight: 700;
	font-size: 16pt;
	padding: 4px;
	border: 1px solid rgb(6, 134, 103);
}

QRadioButton {
	padding: 0px auto;
	font-size: 16pt;
	font-weight: 500;
	background-color: white;
	color: rgb(29, 31, 39);
	border: 1px solid rgb(29, 31, 39);
}

QRadioButton::indicator {
	width: 0;
	height: 0;
}

QRadioButton:hover {
	background-color: rgb(234, 234, 234);
}

QRadioButton:checked {
	color: white;
	background-color: rgb(6, 134, 103);
	border: none;
}

#remarksLabel, #sourceLabel, #destLabel, #typeLabel{
	color:rgb(6, 134, 103);
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>100</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>100</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <property name="spacing">
            <number>30</number>
           </property>
           <item row="0" column="1">
            <widget class="QLabel" name="sourceLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="scheduleInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Format: Second Minute Hour Day Month Weekday</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>Ex. 0 0 12 * * * (Every Day at 12:00:00)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="destLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Remarks</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Source</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="typeLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="remarksLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Destination</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">QLabel {
	font-weight: 700;
	letter-spacing: 0.5px;
}</string>
             </property>
             <property name="text">
              <string>CRON String</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-size: 14pt;
	padding: 8px;
	text-align: center;
}

QPushButton#backButton {
	background: white;
	color: rgb(29, 31, 39);
	border: 1px solid rgb(29, 31, 39);
}

QPushButton#backButton:hover {
	background: rgb(234, 234, 234);
}

QPushButton#backButton:disabled {
	background: lightgrey;
	color: darkgrey;
	border: none;
}

QPushButton#nextButton:disabled {
	background: grey;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	font-size: 14pt;
	padding: 8px;
	text-align: center;
}

QPushButton#backButton {
	background: white;
	color: rgb(29, 31, 39);
	border: 1px solid rgb(29, 31, 39);
}

QPushButton#backButton:hover {
	background: rgb(234, 234, 234);
}

QPushButton#backButton:disabled {
	background: lightgrey;
	color: darkgrey;
	border: none;
}

QPushButton#nextButton:disabled {
	background: grey;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="backButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Create Schedule</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
